src/components/layout/EnterName.js

```diff
@@ -1,16 +1,30 @@
-import React from 'react'
+import { useState } from 'react'
 import ProceedButton from './ProceedButton'
 
-function EnterName() {
+export function EnterName(props) {
+	const [name, setName] = useState("");
+
+	function saveToLocalStorage(event) {
+		setName(event.target.value);
+		 window.localStorage.setItem('userName', event.target.value)
+	}
+
 	return (
 		<>
-			<form className='form__name'>
-            <input type='text' name='name' id='name' />
+			<div className='form__name'>
```

Comment:
Not a huge issue but for semantics it's recommended to use a form tag to surround inputs.

Reply:
Yes, it's true. But if we use <form>, the button at the end of the form will behave like a Submit button and it will reload the page. And we don't want that behaviour, we want the page to transition to the next page without refresh